package.json

```
@@ -2,7 +2,7 @@
   "name": "size-report",
   "version": "0.1.0",
   "description": "Generate size report for pull request",
-  "author": "OpenThread Authors <openthread-users@googlegroups.com>",
+  "author": "OpenThread Authors https://github.com/openthread/openthread/discussions",
```

```suggestion
  "author": "OpenThread Authors (https://github.com/openthread/openthread)",
```
See https://docs.npmjs.com/creating-a-package-json-file#author-field for formatting. We can provide a link, but we can just link to the repo itself. @Vyrastas , thoughts?

---

Yes let's just link to the repo only.